packages/cli/generators/project/templates/.prettierignore

```diff
@@ -1,4 +1,3 @@
 dist
-dist6
```

Comment:
I am not a fan of making unrelated changes. Can you at least split the removal of `dist6` entries into a standalone commit please?